docs/displaying-ads-hook.mdx

```diff
@@ -98,3 +99,5 @@ Return values of the hook are:
 > Note that `isOpened` value remains `true` even after the ad is closed.
 > The value changes to `false` when ad is initialized via calling `load()`.
 > To determine whether the ad is currently showing, use `isShowing` value.
+
+[Impression-level ad revenue]: /impression-level-ad-revenue
```

Comment:
I don't think this is needed here - I think what is needed is an entry in the sidebar - https://github.com/invertase/react-native-google-mobile-ads/blob/03e18fb414c837588fd652fe424cfc15445fdc93/docs.json#L9-L10

Can preview at https://docs.page/invertase/react-native-google-mobile-ads~477 - not sure why the docs.page github bot isn't commenting on the PR to drop that URL in there for preview

Reply:
`[Impression-level ad revenue]: /impression-level-ad-revenue` is needed, because it is used at the line 93. Indeed, I missed to update `docs.json` file, I kind of thought it just automatically creates a page for each mdx file in the folder.

Reply:
ah - okay on the first bit

as for docs.page generating an entry for each page, perhaps that is the default mode without the sidebar defined statically? I'm unsure to be honest, but since we *do* have one defined, now we have to add an entry. Easy easy change at least :-)

Reply:
The sidebar config is updated.